CHANGELOG.md

```diff
@@ -110,7 +109,9 @@ The CHANGELOG entries for the releases listed below this were added retroactivel
 ## [1.0] - 2017-12-20
 *Initial release! A browser including home tile shortcuts.*
 
-[Unreleased]: https://github.com/mozilla-mobile/firefox-tv/compare/v3.0...HEAD
+[Unreleased]: https://github.com/mozilla-mobile/firefox-tv/compare/v3.0.2..HEAD
```

Comment:
This should be (3) ... instead of (2) .. but I can fix that in my version update.